src/PatientChecking/PatientChecking/wwwroot/css/PatientList.css

```diff
@@ -0,0 +1,90 @@
+﻿.search-container {
+    display: flex;
+    align-items: center;
+    justify-content: end;
+    width: 100%;
+    margin: 10px;
+    padding: 10px;
+    font-size: 16px !important;
+}
+
+.search-area {
+    width: 30%;
+}
+
+.sort-area {
+    width: 20%;
+    margin-left: 10px;
+}
+
+.list-patient-container {
+    margin: 10px 20px;
+    font-size: 2rem;
+    justify-content: center;
+    background-color: white;
+}
+
+.row {
+    border: 1px solid #eff0f2;
+    padding: 15px;
+}
+
+.patient-item1 {
+    display: flex;
+}
+
+.patient-name {
+    margin-left: 10px;
+}
+
+.patient-item2 {
+    display: flex;
+    flex-direction: column;
+}
+
+.patient-item3 {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: space-evenly;
+}
+
+    .patient-item3 > div {
```

Comment:
Please adjust indent of this block

Reply:
I believe this indent of block shouble be keeped for readability. But I will apply it in the next feature